radio/src/opentx.cpp

```diff
@@ -260,9 +260,7 @@ void memswap(void * a, void * b, uint8_t size)
 #if defined(PXX2)
 void setDefaultOwnerId()
 {
-  for (uint8_t i = 0; i < PXX2_LEN_REGISTRATION_ID; i++) {
-     g_eeGeneral.ownerRegistrationID[i] = (((uint8_t *)cpu_uid)[4 + i] & 0x3fu);
-  }
+  strAppendUnsigned( g_eeGeneral.ownerRegistrationID, cpu_uid[0], PXX2_LEN_REGISTRATION_ID, 16);
```

Comment:
This will use the ID stored at `0x1FFF7A10` as an address in memory, which is quite risky in fact, as you might read basically anywhere in memory, thus possibly triggering a `Bus Fault`. I believe what you would want is `&cpu_uid[0]` rather than `cpu_uid`.